config/environments/production.rb

```diff
@@ -55,7 +55,7 @@
                            username: ENV["MEMCACHEDCLOUD_USERNAME"], password: ENV["MEMCACHEDCLOUD_PASSWORD"]
                          }]
                        else
-                         :mem_cache_store
+                         :memory_store
```

Comment:
```suggestion
                         :mem_cache_store
```

I think we should keep MemCacheStore